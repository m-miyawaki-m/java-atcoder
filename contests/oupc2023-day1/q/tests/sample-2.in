999999937 8 3
1 314159265 358979323 846264338 327950288
1 419716939 937510582 97494459 230781640
2
1 628620899 862803482 534211706 798214808
1 651328230 664709384 460955058 223172535
1 940812848 111745028 410270193 852110555
2
1 964462294 895493038 196442881 97566593
3 3 344612847 564823378 67831652 712019091
3 7 45648566 923460348 610454326 648213393
1 8 60726024 914127372 458700660 631558817
