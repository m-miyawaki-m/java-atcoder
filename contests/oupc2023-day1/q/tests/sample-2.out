511746849
453803153
685820807
